pylint/test/functional/both_equality_methods.py

```diff
@@ -0,0 +1,26 @@
+# pylint: disable=too-few-public-methods,missing-docstring
```

Comment:
Can we add a test for more complicated cases (with dunder methods being inherited, with dunder methods being added via metaclass)?